colossalai/inference/core/init_model.py

```diff
@@ -0,0 +1,22 @@
+from transformers import BloomForCausalLM, LlamaForCausalLM, PretrainedConfig
+
+from .config import InferenceConfig
+
+_SUPPORTED_MODELS = {
+    "BloomForCausalLM": BloomForCausalLM,
+    "LlamaForCausalLM": LlamaForCausalLM,
+    "LLaMAForCausalLM": LlamaForCausalLM,  # For decapoda-research/llama-*
+}
+
+
+def init_model(inference_config: InferenceConfig, hf_config: PretrainedConfig):
```

Comment:
There are two problems:
1. there is no need to create a file for one function each time
2. again, this is a function wrapper for merely `model.from_pretrained`, which is rather redundant

Reply:
Because we no longer need to initialize the model, I have removed it.